core/src/scene/sceneLoader.cpp

```diff
@@ -744,12 +744,40 @@ void SceneLoader::parseStyleProps(Node styleProps, StyleParamMap& paramMap, cons
         } else if(propItr->second.IsMap()) {
             parseStyleProps(propItr->second, paramMap, paramKey);
         } else {
-            logMsg("Error: Badly formed Style property, need to be a scalar, sequence or map. %s will not be added to stype properties.\n", paramKey.c_str());
+            logMsg("Error: Badly formed Style property, need to be a scalar, sequence or map."
+                    "%s will not be added to stype properties.\n", paramKey.c_str());
         }
     }
 
 }
 
+void SceneLoader::loadSublayers(YAML::Node layer, std::vector<std::shared_ptr<SceneLayer>>& subLayers) {
+
+    for (auto subLayerItr = layer.begin(); subLayerItr != layer.end(); ++subLayerItr ) {
+
+        std::string subLayerName = subLayerItr->first.as<std::string>();
+        if( subLayerName != "data" && subLayerName != "filter" && subLayerName != "draw" &&
```

Comment:
@hjanetzek I agree, that would make things easier. However we're following the syntax that's been established by tangram-js: https://github.com/tangrams/tangram-docs/blob/gh-pages/pages/layers.md#sublayer-name
To keep from needing a list of reserved words we could check if the element has a child called 'draw'